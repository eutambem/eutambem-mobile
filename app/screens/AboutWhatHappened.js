import React from 'react';
import PropTypes from 'prop-types';

import Container from '../components/common/Container';
import AboutWhatHappenedForm from '../components/AboutWhatHappenedForm';

const AboutWhatHappened = props => (
  <Container>
    <AboutWhatHappenedForm navigation={props.navigation} />
  </Container>
);

AboutWhatHappened.navigationOptions = {
  headerTitle: 'Sobre o que aconteceu',
};

AboutWhatHappened.propTypes = {
  navigation: PropTypes.shape({
    navigate: PropTypes.func,
  }).isRequired,
};

export default AboutWhatHappened;
