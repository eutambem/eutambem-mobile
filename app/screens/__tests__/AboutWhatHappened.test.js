import 'react-native';
import React from 'react';
import { shallow } from 'enzyme';
import sinon from 'sinon';

import AboutWhatHappened from '../AboutWhatHappened';

const props = {
  navigation: {
    navigate: sinon.spy(),
  },
};

describe('AboutWhatHappened', () => {
  it('renders correctly', () => {
    const wrapper = shallow(<AboutWhatHappened {...props} />);
    expect(wrapper).toMatchSnapshot();
  });
});
