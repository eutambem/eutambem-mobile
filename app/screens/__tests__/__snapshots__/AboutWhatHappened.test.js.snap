// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AboutWhatHappened renders correctly 1`] = `
<Container
  innerContainerStyle={Object {}}
>
  <_class2
    navigation={
      Object {
        "navigate": [Function],
      }
    }
  />
</Container>
`;
